src/compile/model.ts

```diff
@@ -264,12 +286,24 @@ export abstract class Model {
     return this._transform || {};
   }
 
-  /** Get "field" reference for vega */
-  public field(channel: Channel, opt: FieldRefOption = {}) {
-    const fieldDef = this.fieldDef(channel);
+  /**
+   * Just the raw field. Get's the value from the iterator if the parent is iterating.
+   */
+  public fieldOrig(channel: Channel): string {
```

Comment:
Yes. I added a todo. I think we should use optional argument anyway instead of this option argument.